Rock.Rest/Controllers/FinancialTransactionsController.Partial.cs

```diff
@@ -317,7 +317,33 @@ public DataSet GetContributionTransactions( int groupId, int? personId, [FromBod
         }
 
         /// <summary>
-        /// 
+        /// Gets the contribution transactions.
+        /// </summary>
+        /// <param name="groupId">The group unique identifier.</param>
+        /// <returns></returns>
+        /// <exception cref="System.Web.Http.HttpResponseException"></exception>
+        [Authenticate, Secured]
+        [HttpGet]
+        [EnableQuery]
+        [System.Web.Http.Route( "api/FinancialTransactions/GetTransactionsForGivingGroup/{personId}" )]
+        public IQueryable<FinancialTransaction> GetTransactionsForGivingGroup( int personId )
+        {
```

Comment:
@benwiley , sorry yeah it should use 'GivingId' and that is a string. So method should be `GetByGivingId( string givingId )`

They are similar concepts. If person has selected to give with their family, their GivingGroupId is the id of their family group, and their GivingId is that same id with a 'G' prefix. If they have selected not to give with their family, then their GivingGroupId is null, and their GivingId is their person id with a 'P' prefix. So you could never query by GivingGroupId as many people would have a null value, but different GivingIds.